android-components/components/browser/state/src/test/java/mozilla/components/browser/state/engine/middleware/TranslationsMiddlewareTest.kt

```diff
@@ -239,6 +241,107 @@ class TranslationsMiddlewareTest {
         waitForIdle()
     }
 
+    @Test
+    fun `WHEN UpdatePageSettingAction is dispatched WITH UPDATE_ALWAYS_TRANSLATE_LANGUAGE AND updating the setting is unsuccessful THEN OperationRequestedAction with FETCH_PAGE_SETTINGS is dispatched`() = runTest {
+        // Setup
+        setupMockState()
+        val errorCallback = argumentCaptor<((Throwable) -> Unit)>()
+        whenever(
+            engine.setLanguageSetting(
+                languageCode = any(),
+                languageSetting = any(),
+                onSuccess = any(),
+                onError = errorCallback.capture(),
+            ),
+        )
+            .thenAnswer { errorCallback.value.invoke(Throwable()) }
```

Comment:
nit: remove the indent or move it up one line for better readability?

Reply:
Updated, ty!